src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CoseKeyHelpers.cs

```diff
@@ -0,0 +1,261 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Security.Cryptography;
+
+namespace System.Formats.Cbor.Tests
+{
+    public static class CborCoseKeyHelpers
+    {
+        public static byte[] ExportCosePublicKey(ECDsa ecDsa, HashAlgorithmName hashAlgName)
```

Comment:
Assuming this becomes public API in the future should there be a Span overload?

Reply:
(also the argument names should not be abbreviated)

Reply:
It's more a proof of concept, establishing that the CBOR apis work well enough for the use case we are mostly interested in.

Reply:
that convinces me more this should work with Span... 😄 I'm fine with doing this later but I think it actually might be more work to do it later